### PR TITLE
add post for 2021-12-20 on the Lesson Infrastructure Committee

### PR DESCRIPTION
**Pull request by zkamvar**
- [x] get feedback from interviewees
 - [x] ask for consent to be named from intervewees
 - [x] add relevant links to github repos/discussion threads

### CODE REVIEW

File: _posts/2021/12/2021-12-20-lesson-infrastructure-committee.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+ - markdown was only 2 years old 
+ - git was only 3 years old
+ - github did not exist
+ - jekyll did not exist
+ - smartphones and tablets did not exist
+
+The lesson infrastructure has evolved so much in the past 14 years along with the vastly changing landscape of online publishing where it is now easier than ever to publish a website from a handful of markdown documents.
+
+A large part of my job for the last 20 months has been to learn from the Maintainer community and subsequently understand, analyse, and re-imagine our lesson infrastructure in a centralised way that can serve our Maintainers, Instructors, and Learners equitably and reduce barriers for entry.
```
Comment (k8hertweck) [inline _posts/2021/12/2021-12-20-lesson-infrastructure-committee.md]: I would clarify your role in the Carpentries (I don't think it gets listed in the authorship?)

**Comment by k8hertweck** [comment]
I love the tone of how this is written, and think it'll resonate with folks who have thought lots about lesson infrastructure, and be interesting to people who have not.

It is confusing that the authorship includes Alycia, given the framing of the language. I added a note to specify that "I" means "Zhian" early in the post. I think it is good to include Alycia's community role, and it may be appropriate to reference her involvement later in the post?

I'm happy to be acknowledged (and thus, share responsibility for the historical state of things). I'm glad to see some solid, actionable steps being taken to clarify responsibility for these choices and the technical work, and look forward to seeing where it goes from here!

File: _posts/2021/12/2021-12-20-lesson-infrastructure-committee.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+Our new lesson infrastructure removes much of the temptation for [bikeshedding](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3976103/?report=classic#s7) by completely separating the tools from the lessons themselves. One thing is clear: if we want to ensure that our lesson infrastructure adheres to our core values, we need to have a clear path for community input. At this time, the new infrastructure is not yet in a mature state. It is currently operational and we have determined through testing and feedback that the approach is sound, but the documentaion needs vast improvment and unguided testing from maintainers/lessond developers is still necessary. To have a committee meet to discuss this rapidly-shifting work would be a waste of their valuable time, especially as the pandemic continues to loom and we are still adapting to the ever changing policy landscape and stress it wrought.
```
Comment (naupaka) [inline _posts/2021/12/2021-12-20-lesson-infrastructure-committee.md]: Typo: maintainers/lessond -> maintainers/lesson

**Comment by jduckles** [comment]
Great blog post! Thank you for taking the time to understand the context. I think it places things on a strong foundation for moving forward in 2022. All good from me. 👍🏼

File: _posts/2021/12/2021-12-20-lesson-infrastructure-committee.md (hunk 1 above)
Comment (zkamvar) [reply _posts/2021/12/2021-12-20-lesson-infrastructure-committee.md]: Good point! I'll address that

**Comment by fmichonneau** [comment]
Very well-written blog post! Thank you for synthesizing so eloquently the work from this committee.

File: _posts/2021/12/2021-12-20-lesson-infrastructure-committee.md (hunk 2 above)
Comment (zkamvar) [reply _posts/2021/12/2021-12-20-lesson-infrastructure-committee.md]: good catch! thank you

**Pull request merged**